README.md

```diff
@@ -307,6 +307,11 @@ data already loaded.
   bin/sandbox
   ```
 
+  Depending on your local environment, it may necessary for you to set environment variables for your RDBMS, namely:
```

Comment:
```suggestion
  Depending on your local environment, it may be necessary for you to set environment variables for your RDBMS, namely:
```

Reply:
Ooops, sorry for the typo @waiting-for-dev !